test/e2e/generate_kube_test.go

```diff
@@ -58,7 +58,7 @@ var _ = Describe("Podman generate kube", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 
-		kube := podmanTest.Podman([]string{"generate", "kube", "top"})
+		kube := podmanTest.Podman([]string{"kube", "generate", "top"})
```

Comment:
Are there still instances of `generate kube` left?

Reply:
Yes there is still instances of generate kube left